Jenkinsfile

```diff
@@ -8,7 +8,6 @@ pipeline {
   stages {
     stage('Build') {
       environment {
-        GITHUB_TOKEN = credentials('jenkins_release_token')
```

Comment:
Why isn't GITHUB_TOKEN needed anymore?

Reply:
Because we don't publish the artifact to GitHub in the build step.

Reply:
Ah, I didn't notice it was there in the publish step. 👍